Backend/Tumultu/Infrastructure/DataProviders/Database/EFCore/Write/EfCoreFilesRepository.cs

```diff
@@ -0,0 +1,20 @@
+using Microsoft.EntityFrameworkCore;
+using Tumultu.Application.Files.Commands;
+using Tumultu.Domain.Entities;
+
+namespace Tumultu.Infrastructure.DataProviders.Database.EFCore.Write;
+
+public class EfCoreFilesRepository : EfCoreRepository<FileEntity, Guid>, IFilesRepository
+{
+    public EfCoreFilesRepository(DbContext context) : base(context)
+    {
+    }
+
+    public async Task<IEnumerable<FileEntity>> GetAllByAnySignature(string? md5Signature, string? sha1Signature, string? sha256Signature)
```

Comment:
I wonder, this is in EFCore while being a query, shouldn't we move it into dapper?